inc/api.class.php

```diff
@@ -919,9 +919,12 @@ protected function getItems($itemtype, $params = array(), &$totalcount=0) {
             $range = explode("-", $params['range']);
             $params['start']      = $range[0];
             $params['list_limit'] = $range[1]-$range[0]+1;
+            $params['range']      = $range;
          } else {
             $this->returnError("range must be in format : [start-end] with integers");
          }
+      } else{
+         $params['range'] = array(0,50);
```

Comment:
instead 50, $_SESSION['glpilist_limit'] could be better. It can be set and controlled in the interface